test/Validate.CFG.cpp

```diff
@@ -638,7 +639,8 @@ TEST_F(ValidateCFG, NestedLoops) {
   str += loop1 >> vector<Block>({loop2, loop1_merge});
   str += loop2 >> vector<Block>({loop2, loop2_merge});
   str += loop2_merge >> loop1_merge;
-  str += loop1_merge;
+  str += loop1_merge >> vector<Block>({loop1, exit});
```

Comment:
Which reminds me: we should add a TODO to test that the validator flags this violation. :)